dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/authentication/ResetPasswordResourceIntegrationTest.java

```diff
@@ -84,12 +88,12 @@ public void testTokenInvalidException() throws DotSecurityException, NoSuchUserE
     }
 
     @Test
-    public void testTokenExpiredException() throws DotSecurityException, NoSuchUserException, DotInvalidTokenException {
+    public void testTokenExpiredException() throws DotSecurityException, NoSuchUserException, DotInvalidTokenException, DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)